models/api/course-material/index.js

```diff
@@ -17,6 +17,15 @@ const courseMaterialSchema = new Schema({
         type: ObjectId,
         ref: "BranchLevel",
     }],
+    daySequence: {
+        type: Number,
+        required: true
+    },
+    classType: [{
+        type: String,
+        enum: ["FSJS", "XD","FE"],
```

Comment:
So if we have an assignment that we want to use for both the full stack and frontend class, which of these will we mark is as? As of right now, the Frontend class will just have all the same material as the first 7-8 weeks of the full stack class